src/browser-frontend/views/browser/chrome/tabbar/tab.jsx

```diff
@@ -47,7 +48,12 @@ export default class Tab extends PureComponent {
     this.props.dispatch(PagesModelActions.setSelectedPage({ pageId: this.props.pageId }));
   }
 
+  drag = (e) => {
```

Comment:
Nit: rename this to `handleDragStart` for consistency and following the "idiomatic" style.